packages/turf-nearest-point/main.es.js

```diff
@@ -0,0 +1,50 @@
+import clone from '@turf/clone';
```

Comment:
@rowanwins `main.es.js` accidentally got added to your PR, I'm assuming the `.gitignore` wasn't updated when you created this branch.

Reply:
Yeah you're probably right sorry.